src/particle.h

```diff
@@ -0,0 +1,160 @@
+    double xyz[3]; //!< particle position
+    double uvw[3]; //!< particle direction
+    bool rotated {false};  //!< Is the level rotated?
+
+    //! clear data from a single coordinate level
+    void reset();
+  };
+
+  //============================================================================
+  //! State of a particle being transported through geometry
+  //============================================================================
+
+  struct Particle {
+    int64_t id;  //!< Unique ID
+    int type;    //!< Particle type (n, p, e, etc.)
+
+    int n_coord;                  //!< number of current coordinate levels
+    int cell_instance;            //!< offset for distributed properties
+    LocalCoord coord[MAX_COORD];  //!< coordinates for all levels
+
+    // Particle coordinates before crossing a surface
+    int last_n_coord;          //!< number of current coordinates
+    int last_cell[MAX_COORD];  //!< coordinates for all levels
+
+    // Energy data
+    double E;       //!< post-collision energy in eV
+    double last_E;  //!< pre-collision energy in eV
+    int g;          //!< post-collision energy group (MG only)
+    int last_g;     //!< pre-collision energy group (MG only)
+
+    // Other physical data
+    double wgt;     //!< particle weight
+    double mu;      //!< angle of scatter
+    bool alive;     //!< is particle alive?
+
+    // Other physical data
+    double last_xyz_current[3];  //!< coordinates of the last collision or
+                                 //!< reflective/periodic surface crossing for
+                                 //!< current tallies
+    double last_xyz[3];          //!< previous coordinates
+    double last_uvw[3];          //!< previous direction coordinates
+    double last_wgt;             //!< pre-collision particle weight
+    double absorb_wgt;           //!< weight absorbed for survival biasing
+
+    // What event took place
+    bool fission;       //!< did particle cause implicit fission
+    int event;          //!< scatter, absorption
+    int event_nuclide;  //!< index in nuclides array
+    int event_MT;       //!< reaction MT
+    int delayed_group;  //!< delayed group
+
+    // Post-collision physical data
+    int n_bank;        //!< number of fission sites banked
+    double wgt_bank;   //!< weight of fission sites banked
+    int n_delayed_bank[MAX_DELAYED_GROUPS];  //!< number of delayed fission
+                                             //!< sites banked
+
+    // Indices for various arrays
+    int surface;        //!< index for surface particle is on
+    int cell_born;      //!< index for cell particle was born in
+    int material;       //!< index for current material
+    int last_material;  //!< index for last material
+
+    // Temperature of current cell
+    double sqrtkT;       //!< sqrt(k_Boltzmann * temperature) in eV
+    double last_sqrtkT;  //!< last temperature
+
+    // Statistical data
+    int n_collision;  //!< # of collisions
```

Comment:
Replace the `#` with `number`.  Doxygen interprets the `#` with markdown as a header.